.vitepress/config.ts

```diff
@@ -728,6 +728,10 @@ function sidebarHome() {
                 },
               ],
             },
+            {
```

Comment:
@Mackenzie-OO7 what do you think of putting this in the node operators section in celestia-app category?
<img width="250" alt="screenshot 2024-01-24 at 17 06 55" src="https://github.com/celestiaorg/docs/assets/46639943/99f5320f-42c0-4917-9dbe-20bb833a7252">

Reply:
hii, sorry for the delay. of course! not sure why I put it in the developers section in first place😅

Reply:
no worries! i was about to move it myself and approve PR 😄